src/common/comment.rs

```diff
@@ -0,0 +1,240 @@
+const PREFIX: &str = "dotenv-linter";
+const ON: &str = "on";
+const OFF: &str = "off";
+
+#[derive(Debug, PartialEq)]
+pub struct Comment<'a> {
+    disable: bool,
+    pub checks: Vec<&'a str>,
+}
+
+pub fn parse(s: &str) -> Option<Comment> {
+    // A comment without the # character
+    let line_comment = s.trim_start()[1..].trim();
+
+    if !line_comment.starts_with(PREFIX) {
```

Comment:
This condition seems excess, because in 21 line now exists `strip_prefix` with `?`.
If i am not mistaken it will be enough :relaxed: